File: app/src/main/java/org/ionchain/wallet/mvp/callback/OnImportPrivateKeyCallback.java
package org.ionchain.wallet.mvp.callback;

/**
 * USER: binny
 * DATE: 2018/9/17
 * 描述: 导出私钥
 */
public interface OnImportPrivateKeyCallback {
    /**
     * @param privateKey
     */
    void onImportPriKeySuccess(String privateKey);

    /**
     * @param erroe
     */
    void onImportPriKeyFailure(String erroe);
}

File: app/src/main/java/org/ionchain/wallet/mvp/callback/OnImportPrivateKeyCallback.kt
package org.ionchain.wallet.mvp.callback

/**
 * USER: binny
 * DATE: 2018/9/17
 * 描述: 导出私钥
 */
interface OnImportPrivateKeyCallback {
    /**
     * @param privateKey
     */
    fun onImportPriKeySuccess(privateKey: String)

    /**
     * @param erroe
     */
    fun onImportPriKeyFailure(erroe: String)
}
